pxtlib/tilemap.ts

```diff
@@ -816,6 +816,9 @@ namespace pxt {
                 case AssetType.Image:
                     return this.state.images.getByID(name) || this.gallery.images.getByID(name);
                 case AssetType.Tile:
+                    if (name == "myTiles.tile0") {
+                        return this.getTransparency(16);
```

Comment:
this should probably be the tilewidth instead of 16

Reply:
ahhh yes agreed. I was having issues with tilewidths - are all the tilemaps in a tilemapProject supposed to have the same tilewidth?

Reply:
no, they can have different widths

Reply:
oh okay hmmmm i need to fix some things